_posts/2024-08-01-given-priority-change-3.5.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: blog-detail
+post-type: blog
+by: Oliver Bračevac, EPFL
+title: "Changes to Givens in Scala 3.5"
+---
+
+## New Prioritization of Givens
+
+Starting with Scala 3.5, the prioritization of givens has been
```

Comment:
I think the correct version for prioritization of givens changes is 3.6, in 3.5 We just warming it up with warning.

Reply:
We were considering switching it on for 3.5 already. In the end, we decided against it to give people more time to test it.